roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -259,7 +352,7 @@ def load_checks(path=None, subpkg=""):
             modules = modules + load_checks(os.path.join(path, name), subpkg + "." + name)
             continue
 
-        if name.endswith(".py") and not name.startswith(".") and name not in LOADER_EXCLUDES:
+        if name.endswith(".py") and name not in LOADER_EXCLUDES:
```

Comment:
@sosiouxme, somehow I missed this during review.  Why remove the hidden-file check?

Reply:
No good reason I can think of; probably a mistake during fixing a merge conflict